src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.cs

```diff
@@ -0,0 +1,68 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Diagnostics.Tracing;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Text;
+
+[assembly: System.Resources.NeutralResourcesLanguage("en-us")]
+
+namespace System.Text.RegularExpressions.Generator
+{
+    /// <summary>Generates C# source code to implement regular expressions.</summary>
+    [Generator]
```

Comment:
Might be a good idea to explicitly supply `LanguageNames.CSharp` to the attribute. It's currently the same, but just in case roslyn changed that in future 😄